src/components/flat-table/flat-table-checkbox/__snapshots__/flat-table-checkbox.spec.js.snap

```diff
@@ -180,7 +174,6 @@ exports[`FlatTableCheckbox "th" is passed in via the "as" prop renders to match
               aria-labelledby="guid-12345-label"
               className="c10 c11"
               id="guid-12345"
-              name="flat-table-checkbox"
```

Comment:
`name` prop is not passed to `input` anymore, that's not good :)

Reply:
ahh, you are right. Changed, thanks